suites/squid/nvmeof/tier-2_nvmeof_b2b_upgrade.yaml

```diff
@@ -34,8 +34,8 @@ tests:
               command: bootstrap
               service: cephadm
               args:
-                custom_repo: https://public.dhe.ibm.com/ibmdl/export/pub/storage/ceph/testing/IBM-CEPH-8.0-202408281222.ci.0/IBM-CEPH-8.0-202408281222.ci.0-rhel9.repo
-                custom_image: cp.stg.icr.io/cp/ibm-ceph/ceph-8-rhel9:8-25
+                custom_repo: https://public.dhe.ibm.com/ibmdl/export/pub/storage/ceph/testing/IBM-CEPH-8.0-202409180241.ci.0/IBM-CEPH-8.0-202409180241.ci.0-rhel9.repo
```

Comment:
@HaruChebrolu Is b2b upgrade supported from "cp.stg.icr.io/cp/ibm-ceph/ceph-8-rhel9:8-30" ?

Reply:
Thats the version mentioned in mail for that specific IBM build snapshot
```
You can find the RHEL repos here:

https://public.dhe.ibm.com/ibmdl/export/pub/storage/ceph/testing/IBM-CEPH-8.0-202409180241.ci.0/

Containers in IBM's Staging registry:

  cp.stg.icr.io/cp/ibm-ceph/ceph-8-rhel9:8-30
  cp.stg.icr.io/cp/ibm-ceph/cephcsi:v4.16.0-33
  cp.stg.icr.io/cp/ibm-ceph/grafana-rhel9:10.4.0-15
  cp.stg.icr.io/cp/ibm-ceph/haproxy-rhel9:2.4.22-29
  cp.stg.icr.io/cp/ibm-ceph/keepalived-rhel9:2.2.8-28
  cp.stg.icr.io/cp/ibm-ceph/logging-loki-rhel8:v2.6.1-51
  cp.stg.icr.io/cp/ibm-ceph/nginx-124-rhel9:1-25.1725906898
  cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.3.2-1
  cp.stg.icr.io/cp/ibm-ceph/nvmeof-rhel9:1.3.2-1
```